eng/ci_tools.txt

```diff
@@ -24,10 +24,8 @@ coverage==7.2.5
 Jinja2==3.1.2
 MarkupSafe==2.1.3
 json-delta==2.0
-readme_renderer==37.3; python_version <= '3.7'
 readme_renderer==42.0; python_version > '3.7'
```

Comment:
We can remove python_version > '3.7' too? :)

Reply:
Sure. I'll do it.